nengo/base.py

```diff
@@ -330,6 +359,10 @@ def make_step(self, shape_in, shape_out, dt, rng):
             The simulation timestep.
         rng : `numpy.random.RandomState`
             A random number generator.
+        state : {string: `numpy.ndarray`}
```

Comment:
Above in `allocate` the docstring says `initial_state : {string: .builder.Signal}` instead of `numpy.ndarray`. Is this intentional?

Reply:
No, they should both be ```{string: `numpy.ndarray`}```. The output of `allocate` is turned into `Signal`s in the builder, but is arrays.